test/controllers/name_controller_test.rb

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hmm. Because utf-8 is Ruby's default encoding, I've been omitting (and sometimes removing) similar magic comments in other files.
Should I add/restore them?